src/sql/base/browser/ui/table/plugins/rowMoveManager.plugin.ts

```diff
@@ -0,0 +1,169 @@
+// Adopted and converted to typescript from https://github.com/mleibman/SlickGrid/blob/gh-pages/plugins/slick.rowmovemanager.js
+// heavily modified
+
+// eslint-disable-next-line header/header
```

Comment:
what is this for?

Reply:
the linting complains about the copyright header but this isnt our file

Reply:
Add to https://github.com/microsoft/azuredatastudio/blob/c56f035a06e242bcb0c5a9b0db6b8f7c667a4b67/build/filters.js#L128 instead

Reply:
@Charles-Gagnon I added this file to the filter but it's still seeing a hygiene issue. What am I missing here?

Reply:
Fixed, needed to add it to eslintrc as well